PostBy/Global Variables/GlobalVars.m

```diff
@@ -32,3 +32,5 @@
 NSString *const LIKES_RELATION = @"likes";
 NSString *const DISLIKES_RELATION = @"dislikes";
 
+// !IMPORTANT: Any changes made to fields here will need to be manually done in Parse Server's Background Jobs
```

Comment:
This warning is important but with ambiguity. 

Does "here" mean the fields above this line or below this line? If this file has thousands of lines and a developer only updates the first few lines, does the developer have the chance to notice this warning?

Reply:
Good point Rao! I will put the comment at the top of the file and specific "fields below"